reco_utils/recommender/deeprec/models/sequential/nextitnet.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import tensorflow as tf
+from reco_utils.recommender.deeprec.models.sequential.sequential_base_model import (
+    SequentialBaseModel,
+)
+
+__all__ = ["NextItNetModel"]
+
+
+class NextItNetModel(SequentialBaseModel):
```

Comment:
Please add some test cases for the new added model

Reply:
Added test cases in test_deeprec_models.py